Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -45,927 +47,830 @@
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName(this.criticalityPropertyName)
+                        .withTitle("Kritikalitet")
+                        .withId("criticality")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                        .withFixedValueRange(
+                            Kitos.Helpers.KendoOverviewHelper.mapDataForKendoDropdown(
+                                this.criticalityOptionViewModel.options,
+                                true),
+                            false)
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderString(
+                            this.criticalityOptionViewModel.getOptionText(dataItem.Criticality?.Id)))
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderString(
+                            this.criticalityOptionViewModel.getOptionText(dataItem.Criticality?.Id))))
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName("ResponsibleOrganizationUnit.Name")
+                        .withTitle("Ansvarlig org. enhed")
+                        .withId("responsibleOrganizationUnitName")
+                        .withSourceValueEchoRendering()
+                        .withSourceValueEchoExcelOutput()
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                        .withFixedValueRange(orgUnits.map((unit) => {
+                                return {
+                                    textValue: unit.Name,
+                                    remoteValue: unit.Id,
+                                    optionalContext: unit
+                                };
+                            }),
+                            false,
+                            dataItem => "&nbsp;&nbsp;&nbsp;&nbsp;".repeat(dataItem.optionalContext.$level) +
+                            dataItem.optionalContext.Name)
+                        .withRendering(
+                            dataItem => Helpers.RenderFieldsHelper.renderString(dataItem
+                                .ResponsibleOrganizationUnit?.Name))
+                        .withExcelOutput(
+                            dataItem => Helpers.ExcelExportHelper.renderString(dataItem
+                                .ResponsibleOrganizationUnit?.Name)))
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName("Supplier.Name")
+                        .withTitle("Leverandør")
+                        .withId("supplierName")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderString(dataItem.Supplier?.Name))
```

Comment:
I tried using the "withSourceValueEchoRendering" method, but the value wouldn't show